### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] Edit field: Label not read.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11283]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/textarea
When the textarea, only the placeholder value is mentioned and not its label by screen reader.
The corresponding label should be read out, when each & every UI is accessed.
#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use textarea and few other input with place holder and label. Label won't be read by screen reader

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide the form example you are testing and the history from JAWS?

**Comment by JyotiKumari1507** [comment]
@droshev  Accessibility team found that screen reader is not able to read textarea label while testing. We can have more discussion in next call

**Comment by droshev** [comment]
@JyotiKumari1507 we need the input written so it will be easier to process

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide a stackblitz example and can you specify what is needed to be set on the component and why can't you set it?

**Comment by JyotiKumari1507** [comment]
@divyamalhotra488 pls provide input

**Comment by droshev** [comment]
@JyotiKumari1507 We didn't get updates for some time so we are closing the issue. If you think there is a bug, please, create a stackblitz example, document the steps and what the expectation is in a new ticket.

**Comment by divyamalhotra488** [comment]
Created new defect https://github.com/SAP/fundamental-ngx/issues/11965
